float64 ir_back
float64 ir_front
float64 sonar1
float64 sonar2